recipes/foonathan-memory/all/conanfile.py

```diff
@@ -46,10 +46,11 @@ def configure(self):
             del self.options.fPIC
 
     def validate(self):
-        # FIXME: jenkins servers throw error with this combination
+        # FIXME: jenkins servers throw error with this combination 
         # quick fix until somebody can reproduce
         if hasattr(self, "settings_build") and cross_building(self):
-            raise ConanInvalidConfiguration("Cross building is not yet supported. Contributions are welcome")
+            if self.version == '0.7.0' or not (self.settings.os == "Macos" and self.settings.arch != self.settings_build.arch):
```

Comment:
What does it mean, only version 0.7.0 is broken? Because the message says it's not supported for any version.